modules/ingestor/src/service/advisory/osv/loader.rs

```diff
@@ -22,12 +24,15 @@ impl<'g> OsvLoader<'g> {
     pub async fn load<L: Into<String>, R: Read>(
         &self,
         location: L,
+        issuser: Option<String>,
```

Comment:
You're making us look bad compared to AI.

```suggestion
        issuer: Option<String>,
```